neuralprophet/forecaster.py

```diff
@@ -1444,7 +1485,12 @@ def make_future_dataframe(self, df, events_df=None, regressors_df=None, periods=
         )
         return df_future
 
-    def handle_negative_values(self, df, handle="remove", columns=None):
+    def handle_negative_values(
+        self,
+        df: pd.DataFrame,
+        handle: Union[str, int, float, None] = "remove",
```

Comment:
`handle`  could be assigned 4 different types. Would be interested if there are best practices to simplify that piece of code.

Reply:
We could wrap them in a custom type, not sure if that improves readability though... would leave it as this. Question is rather why do we accept that many different types in the first place.